src/utils.js

```diff
@@ -0,0 +1,48 @@
+
+/**
+ * @param {String} prefix
+ * @param {Array<*>} args
+ * @returns {Array<*>}
+ */
+export function appendPrefix(prefix, args) {
+    if (!prefix) {
+        return args;
+    }
+    if (typeof args[0] === 'string') {
```

Comment:
It's for consistency with `replacePlaceholders`, `format` and `write` methods.
`formatter.format` can return whatever `writer.write` can handle, but all included writers and formatters work with arrays. This arrays can be of two 'types': array of raw log arguments or array one one formatted argument as a string.